### ISSUE TITLE
Squid for Windows 4.14 not working on Windows Server 2019

### ISSUE THREAD

**Issue by stupidcat712** [diladele/squid-windows#95]
Windows Event Log shows "Cannot find a squid process. Trying to start it..." and clients can't access the internet though proxy.
However, Squid for Windows 3.5.28 works perfectly.

**Comment by chmuc01** [comment]
Some here on Windows 10 (20H2). 
Squid service starts but noting works and no access.log or error.log is written.
Trying to start "squid.exe" ends in
squid 1905 cygwin_exception::open_stackdumpfile: Dumping stack trace to squid.exe.stackdump

Seems that there is a bug in tis version.

Also the Squid Tray always pops up a message that a new version 4.14 is available, even if I am just using this version.
Please fix tis and meanwhile make the last working version available for download.

**Comment by ra-at-diladele-com** [comment]
We have tested the install on fresh Windows 10 and everything was fine. Maybe you have some other squid.conf though, could you attach it to this issue? Be sure to remove all persoal info though.

**Comment by stupidcat712** [comment]
My Windows Server 2019 is fresh install and there is no other squid.conf

**Comment by chmuc01** [comment]
Yes, direct after installation the software my work, but as soon as you change the squid.config and/or restart the service, no access.log or error.log is written any more and the service doesn‘t work any more, even if it states that the servos is running.

**Comment by rocx1900** [comment]
same here.. installed 4.14 on windows 10.. no squid process listening on 3128... installed 3.5.28 process is there and listening.. No changes to conf file.. (for both installs)

**Comment by user72356** [comment]
Same here on Windows Server 2016. Service process starts but nothing listening on port 3128 according to netstat.

**Comment by mudwiggle-farm** [comment]
I'm seeing the same cygwin_exception::open_stackdumpfile: Dumping stack trace to squid.exe.stackdump error on Windows 2012 R2
Error happens on a VM on a ESXi 6.7 platform with Sandy Bridge CPU's with the base conf file and when the VM was migrated to another ESXi 6.7 platform with Haswell CPU's it worked without issue, even worked when I changed to my 3.5 conf file
At a guess the code was compiled to to take advantage of a CPU feature that isn't available in some older CPU's

**Comment by darkness-ai** [comment]
Like @mudwiggle-farm said, this seems like an issue of using newer opcodes not available on older CPUs or on some virtualization platforms.

Squid default configure is known to have this issue (as for example stated at - https://unix.stackexchange.com/questions/375696/squid-illegal-instruction-core-dumped)

The fix should be reconfiguring with "--disable-arch-native" and recompiling.

**Comment by darkmyst30** [comment]
I had this same issue on Windows Server 2019 (clean install).  Only got it running for about an hour (no idea how, no changes were made).  It wouldn't open a listening port.

**Comment by dredkin** [comment]
This is ridiculous. Two year already, and the latest release still just cannot start, and nobody cares. I guess I will opt out in favor of another project.

**Comment by coffeecodeconverter** [comment]
for anyone still needing this
check my comment (coffeecodeconverter) on this: https://github.com/diladele/squid-windows/issues/110 thread

FYI im running Windows 11 home, 21H2 build 22000.318 64 bit
Squid version 4.14 - obtained from here: https://squid.diladele.com/
ive also got the same version of squid running on another machine windows 11 home 23h2 build 22621.3527 64 bit

heres a screenshot of the top of my comment, to make it easier to spot scrolling down the page
(my comment is massive btw, Ive provided a full download, install, config, and troubleshooting guidance) 
![image](https://github.com/user-attachments/assets/2e02fede-38d3-4cdf-b0d6-5209487fcadb)

**Comment by thewriteway** [comment]
for those looking newer Squid V6.12 MSI build:
https://github.com/diladele/squid-windows/issues/110#issuecomment-2565352924
